module/x/gravity/keeper/genesis_test.go

```diff
@@ -132,13 +137,61 @@ func TestBatchAndTxImportExport(t *testing.T) {
 		ctx.Logger().Info(fmt.Sprintf("Created batch %v for contract %v with %v transactions", i, v.GetAddress(), batchSize))
 	}
 
-	checkAllTransactionsExist(t, input.GravityKeeper, ctx, txs)
+	// CREATE FORWARDS
```

Comment:
Added ibc auto forwards to our genesis testing, and discovered a bug in our InitGenesis ordering.